Cargo.toml

```diff
@@ -7,18 +7,21 @@ edition = "2018"
 [dependencies]
 futures = "0.1.25"
 http = "0.1"
+http-body = "0.1"
 http-connection = { git = "https://github.com/hyperium/http-connection" }
 log = "0.4"
 hyper = "0.12.28"
 tokio-io = "0.1"
 tokio-buf = "0.1"
 tokio-executor = "0.1"
 tower-service = "0.2"
-tower = { git = "https://github.com/tower-rs/tower" }
-tower-http = { git = "https://github.com/tower-rs/tower-http" }
+tower-util = "0.1"
+# tower-http-util = { git = "https://github.com/tower-rs/tower-http" }
```

Comment:
Leaving this as a TODO to remove on merge of https://github.com/tower-rs/tower-http/pull/23.